python/semantic_kernel/connectors/ai/open_ai/settings/open_ai_settings.py

```diff
@@ -0,0 +1,42 @@
+# Copyright (c) Microsoft. All rights reserved.
+
+from pydantic import SecretStr
+from pydantic_settings import BaseSettings
+
+
+class OpenAISettings(BaseSettings):
```

Comment:
could we make the memory_settings_base even more universal and also use that here?

Reply:
Will handle in a subsequent PR.